components/ui/edit-on-github-button.tsx

```diff
@@ -0,0 +1,20 @@
+import { PencilIcon } from 'lucide-react';
+import { cn } from '@/utils/cn';
+import { buttonVariants } from '@/components/ui/button';
+
+interface EditOnGithubButtonProps {
+  path: string;
+}
+
+export default function EditOnGithubButton({ path }: EditOnGithubButtonProps) {
+  return (
+    <a
+      href={`https://github.com/ava-labs/avalanche-docs/blob/master/${path}`}
```

Comment:
Take the user directly to the edit screen

```suggestion
      href={`https://github.com/ava-labs/avalanche-docs/edit/master/${path}`}
```